evmos/chain.json

```diff
@@ -36,25 +36,22 @@
   },
   "codebase": {
     "git_repo": "https://github.com/evmos/evmos",
-    "recommended_version": "v16.0.3",
+    "recommended_version": "v17.0.0",
     "compatible_versions": [
-      "v16.0.0",
-      "v16.0.1",
-      "v16.0.2",
-      "v16.0.3"
+      "v17.0.0"
     ],
-    "cosmos_sdk_version": "v0.47.5-evmos.2",
+    "cosmos_sdk_version": "evmos/cosmos-sdk v0.47.8-evmos",
```

Comment:
This actually remained unchanged, it's still v0.47.5-evmos.2: https://github.com/evmos/evmos/blob/v17.0.0/go.mod#L253

Reply:
Noted, will fix this one.